third_party/move/move-model/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -212,6 +254,12 @@ impl StacklessControlFlowGraph {
         self.blocks.keys().cloned().collect()
     }
 
+    /// Iterates over all blocks in DFS order starting from the entry block
+    /// `visit_all`: whether to visit all blocks or just blocks reachable from the entry block
+    pub fn iter_dfs_left(&self, visit_all: bool) -> DFSLeft {
```

Comment:
Why is this called "dfs_left"?  Is that "left" as in "remaining"?  Or "left" versus "right"?

Reply:
Ok, I see it below, should be documented here.

Reply:
?

Reply:
Documented